gpflow/kernels.py

```diff
@@ -266,7 +273,35 @@ class RBF(Stationary):
     def K(self, X, X2=None, presliced=False):
         if not presliced:
             X, X2 = self._slice(X, X2)
-        return self.variance * tf.exp(-self.square_dist(X, X2) / 2)
+        return self.variance * tf.exp(-self.scaled_square_dist(X, X2) / 2)
+
+SquaredExponential = RBF
+
+
+class RationalQuadratic(Stationary):
+    """
+    Rational Quadratic kernel,
+
+    k(r) = σ² (1 + r² / 2αℓ²)^(-α)
+
+    σ² : variance
+    ℓ  : lengthscales
+    α  : alpha, determines relative weighting of small-scale and large-scale fluctuations
+
+    For α→ ∞, the RQ kernel becomes equivalent to the squared exponential.
```

Comment:
Add space after alpha. Very nitpicky, but the look of the lack of space makes me sqirm :)

Also, I would do it myself, but then my review would be attached to an old commit.

Reply:
Funnily enough, there is no space *after* the \alpha either... it's just a quirk of the font :roll_eyes: but I'm happy to add a space beforehand if that makes you happier